app/components/Nav/Mobile/navMobile.css

```diff
@@ -0,0 +1,59 @@
+@media only screen and (max-width: 640px) {
+  .top-header {
+    padding: 0;
+  }
+  .top-nav {
+    justify-content: space-between;
+    padding: var(--spacing-20) var(--spacing-24);
+  }
+  .top-logo {
+    padding: 0;
+  }
+  .menu-button {
+    cursor: pointer;
+  }
+  .composite-elements {
+    display: flex;
+    align-items: center;
+  }
+  .space-between {
+    justify-content: space-between;
+  }
+  .flex-start {
+    justify-content: flex-start;
+  }
+  .mobile-menu > .button {
+    border-radius: 0;
+    font-size: 16px;
```

Comment:
never write any CSS for fonts! Assign whichever is the right class based on the Figma, or just h1 or h2 tag.